Makefile

```
@@ -216,3 +216,20 @@ new-gm-data-migration:
 		exit 1; \
 	fi
 	mkdir -p data_migrations/graphqlmetrics/$(shell date +%s)_$(name)
+
+GRAPHQL_GO_TOOLS := github.com/wundergraph/graphql-go-tools/v2
+
+# Usage: make pin-graphql-go-tools BRANCH=<branch-name>
+.PHONY: pin-graphql-go-tools
+pin-engine:
```

checkmake (0.3.2)

[warning] Target body for "pin-engine" exceeds allowed length of 5 lines (6).

(maxbodylength)